module/surveys/helper.js

```diff
@@ -1550,6 +1552,10 @@ module.exports = class SurveysHelper {
           } else {
             // let solutionData = solutionDocument[0];
             const solutionsHelper = require(MODULES_BASE_PATH + '/solutions/helper');
+            if(solutionDocument[0].isAPrivateProgram){
+              solutionDocument[0].referenceFrom = messageConstants.common.PRIVATE;
```

Comment:
@borkarsaish65 what is referenceFrom use here/

Reply:
this is done to skip the scope check since private solution will not have a scope right hence this code was needed